robo_soccer_platform/lib/robo_soccer_platform_web/live/game_dashboard.ex

```diff
@@ -38,8 +38,16 @@ defmodule RoboSoccerPlatformWeb.GameDashboard do
     {room_code, steering_state, game_state} =
       RoboSoccerPlatform.GameController.init_game_dashboard(self())
 
+    config = Application.fetch_env!(:robo_soccer_platform, RoboSoccerPlatformWeb.GameDashboard)
+
+    wifi_qr_svg = render_wifi_qr_code(config[:wifi_ssid], config[:wifi_psk])
+    player_url_qr_svg = render_player_url_qr_code(config[:ip], config[:port], room_code)
+
     socket
+    |> assign(config)
```

Comment:
I think it could be better to separetaly assign `ip`, `port`, `wifi_ssid` and `wifi_psk` here instead of hiding it inside `assign(config)`